tests/test_ingestion.py

```diff
@@ -401,7 +403,19 @@ def testCalibIngestDriver(self):
         """Test that ingesting calibrations starting from an abstract dataset adds them to a repository.
         """
         self.task._ensureRaws(processes=1)  # Should not affect calibs, but would be run
-        self.assertIngestedDataFiles(self.calibData, self.dataset.instrument.makeCollectionName("calib"))
+        self.assertIngestedDataFiles(
+            self.calibData,
+            # queryDatasets cannot (yet) search CALIBRATION collections, so we
+            # instead search the RUN-type collections that calibrations are
+            # ingested into first before being associated with a validity
+            # range.
+            list(
+                self.butler.registry.queryCollections(
+                    re.compile(re.escape(self.dataset.instrument.makeCollectionName("calib") + "/") + ".+"),
+                    collectionTypes={CollectionType.RUN},
+                )
```

Comment:
Can this whole block go into another line, or even a dedicated function? It's very hard to read as it is.